<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <node pkg="ublox_serial" type="ublox_gps_pub" name="pub_gps_data" output="screen"/>
    <node pkg="ublox_serial" type="ublox_gps_sub" name="sub_gps_data" output="screen"/>
</launch>
